### ISSUE TITLE
Change login module to use gh-get-token

### ISSUE THREAD

**Issue by icyflame** [icyflame/node-cube-cli-timer#23]
First, fix https://github.com/icyflame/node-gh-get-token/issues/1

Then use http://npmjs.com/package/gh-get-token

**Comment by Saurabh3333** [comment]
After using gh-get-token for login, I am getting:
`Existing token returned!`
`Visit https://github.com/settings/developers in your browser to revoke access.`

Is it what we are expecting?

**Comment by icyflame** [comment]
Yeah, if there is an existing token for this application, gh-get-token will return the same token. That's how the GitHub api works. (You may have to activate 2 factor authentication to test this feature properly)
